Documentation/Signoffs/Sensors.md

```diff
@@ -0,0 +1,145 @@
+# Sensors System 
+
+  
+
+## Subsystem Function  
+
+  
+
+The role of the sensor subsystem is to send signals—communicating the status of the device occupancy of the presently forefront compartment sections on all three dial layers—to the PLC to update certain tags, so that the PLC's ladder logic code can properly use that information ad hoc.   
+
+  
+
+ 
+
+## Constraints  
+
+  
+
+**The Sensor System:** 
+
+   
+
+* **Cannot take up more than three I/O pins from the PLC**    
+
+   ^The PLC module will have a limited amount of inputs and outputs to receive or send signals. All other pins will need to be allocated to receive from and or send signals to the locks, LEDs, and motor.  
+
+   ^This also makes the constraint that the sensors must send one bit signals since there will have to be three separate —one for each dial layer (which each take up one I/O pin). 
+
+ 
+
+* **Must send a zero signal to the PLC if there is a device in the presently forefront compartment section and a one signal if there is not.**  
```

Comment:
Think about how this will fail. Consider what happens if the wire gets cut. Would you rather the sensor defaults to there's something here or nothing here?